### ISSUE TITLE
Change the size of a fill-handle

### ISSUE THREAD

**Issue by krzysztofspilka** [handsontable/handsontable#4823]
### Description

Let's change the size and style of a fill handle in Handsontable. Right now it is too small and its white border seems to be a bit too wide.

**Current styles:**
```
width: 5px;
height: 5px;
border: 2px solid rgb(255, 255, 255);
```
![screen shot 2018-02-20 at 11 59 55](https://user-images.githubusercontent.com/8048526/36420504-95333aa4-1635-11e8-96de-87451bdadd7c.png)

**Suggested change:**
```
width: 6px;
height: 6px;
border: 1px solid rgb(255, 255, 255);
```
![screen shot 2018-02-20 at 11 54 12](https://user-images.githubusercontent.com/8048526/36420489-8c93721a-1635-11e8-9430-0085b3b7a335.png)

**Comment by pnowak** [comment]
Duplicate #4921

**Comment by AMBudnik** [comment]
Released: v2.0.0
